roles/confluent.ssl/tasks/certificate_authority.yml

```diff
@@ -1,22 +1,22 @@
----
-- name: Create CA
-  shell: |
-    openssl req -new -x509 \
-      -keyout /var/ssl/private/generation/{{ssl_self_signed_ca_key_filename}} \
-      -out /var/ssl/private/generation/{{ssl_self_signed_ca_cert_filename}} \
-      -days 365 \
-      -subj '/CN=ca1.test.confluent.io/OU=TEST/O=CONFLUENT/L=PaloAlto/S=Ca/C=US' \
-      -passin pass:{{ssl_self_signed_ca_password}} -passout pass:{{ssl_self_signed_ca_password}}
-
-- name: Create Truststore and Import the CA Cert
-  shell: |
-    keytool -noprompt -keystore /var/ssl/private/generation/truststore.jks -storetype pkcs12 \
-      -alias CARoot -import -file /var/ssl/private/generation/{{ssl_self_signed_ca_cert_filename}} \
-      -storepass {{truststore_storepass}} -keypass {{truststore_storepass}}
-
-- name: Copy CA Back to Ansible Host
-  synchronize:
-    src: /var/ssl/private/generation/
-    dest: generated_ssl_files
-    mode: pull
-    use_ssh_args: yes
+# ---
+# - name: Create CA
+#   shell: |
+#     openssl req -new -x509 \
+#       -keyout /var/ssl/private/generation/{{ssl_self_signed_ca_key_filename}} \
+#       -out /var/ssl/private/generation/{{ssl_self_signed_ca_cert_filename}} \
+#       -days 365 \
+#       -subj '/CN=ca1.test.confluent.io/OU=TEST/O=CONFLUENT/L=PaloAlto/S=Ca/C=US' \
+#       -passin pass:{{ssl_self_signed_ca_password}} -passout pass:{{ssl_self_signed_ca_password}}
+#
+# - name: Create Truststore and Import the CA Cert
```

Comment:
@domenicbove Should this not be removed if we have moved it into another file now?